core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java

```diff
@@ -32,6 +32,12 @@
 	@Parameter(description = "The types of resources to get the versions for")
 	private List<String> resourceType;
 	
+	@Parameter(description = "Greater than equal to filter for the created at field")
+	private Long createdAtFrom;
```

Comment:
Could you please also add the simple exact match query parameter `createdAt`?

Reply:
Yes

Reply:
Done